ee/codegen/src/__test__/nodes/generic-nodes/__snapshots__/generic-node.test.ts.snap

```diff
@@ -372,6 +374,25 @@ class MyCustomNode(BaseNode):
 "
 `;
 
+exports[`GenericNode > coalesce expression with null RHS > should wrap null RHS in ConstantValueReference 1`] = `
+"from vellum.workflows import BaseNode
+from vellum.workflows.references.constant import ConstantValueReference
+from vellum.workflows.types.core import MergeBehavior
+
+
+class TestCoalesceNullRhsNode(BaseNode):
+    coalesce_attribute = ConstantValueReference("primary_value").coalesce(
+        ConstantValueReference(None)
+    )
```

Comment:
DevinAI this is not emblematic of the issue we saw in production. there was an empty reference on the _left hand side_, such that our codegen on main produced `None.coalesce("value")`, resulting in a NPE